esmvaltool/cmorizers/data/datasets.yml

```diff
@@ -0,0 +1,982 @@
+
+  AURA-TES:
+    tier: 3
+    source: https://search.earthdata.nasa.gov/search?q=TL3O3M
+    last_access: 2018-12-08
+    info: |
+      Download and processing instructions
+      Select the V004 and V005 projects.
+      Download the script file for both projects.
+      To download the data use: wget -i <scriptfile>.
+      From the downloaded data, pick only the *.he5 files and put them in
+      input_dir_path.
+      Data is freely available, but a registration is required.
+
+  BerkeleyEarth:
+    tier: 2
+    source: http://berkeleyearth.org/data/
+    last_access: 2020-02-25
+    info: |
+      Download the following file:
+      http://berkeleyearth.lbl.gov/auto/Global/Gridded/Land_and_Ocean_LatLong1.nc
+
+  CALIPSO-GOCCP:
+    tier: 2
+    source: ftp://ftp.climserv.ipsl.polytechnique.fr/
+    last_access: 2020-01-27
+    info: |
+      Download the data from:"
+        ftp://ftp.climserv.ipsl.polytechnique.fr/
+        path: /cfmip/GOCCP_v3/3D_CloudFraction/grid_2x2xL40/yyyy/avg/
+      Put all files under a single directory (no subdirectories with years).
+        3D_CloudFraction330m_yyyymm_avg_CFMIP2_sat_3.1.2.nc
+
+  CDS-SATELLITE-ALBEDO:
+    tier: 3
+    source: https://cds.climate.copernicus.eu/cdsapp#!/dataset/satellite-albedo?tab=form
+    last_access: 2019-04-01
+    info: |
+      You will need to accept the license prior to use the automatic download.
+      If not, it will fail but will provide you the link to do so.
+
+      Download and processing instructions
+        - Download the data from source to the right directory
+          using the download script.
+        - Decompress the files within the directory:
+            "find . -name '*.tar.gz' -execdir tar -xzvf '{}' \"
+
+  CDS-SATELLITE-LAI-FAPAR:
+    tier: 3
+    source: https://cds.climate.copernicus.eu/cdsapp#!/dataset/satellite-lai-fapar?tab=form
+    last_access: 2019-07-03
+    info: |
+      You will need to accept the license prior to use the automatic download.
+      If not, it will fail but will provide you the link to do so.
+
+      Download and processing instructions
+      - Open in a browser the data source as specified above
+      - Put the right ticks:
+        - Tick variables LAI and FAPAR
+        - Tick satellite SPOT (System Pour l'Observation de la Terre)
+        - Tick sensor VGT (Vegetation)
+        - Tick horizontal resolution 1km
+        - Tick product version V1
+        - Tick all available years
+        - Tick all available months
+        - Tick Nominal day 20
+      - Click 'submit form'
+      - According to ESMValTool practice, put them in the right rawobsdir folder
+      - Request might need to be split into chunks to not exceed download limit
+      - Fails setting standard name for variable FAPAR
+
+  CDS-SATELLITE-SOIL-MOISTURE:
+    tier: 3
+    source: https://cds.climate.copernicus.eu/cdsapp#!/dataset/satellite-soil-moisture?tab=form
+    last_access: 2019-03-14
```

Comment:
The download instructions are missing (this is the only case). I need  to add to the documentation that download instructions from headers of formatter scripts should be copied to this file.